### PR TITLE
update swift

### PR DESCRIPTION
**Pull request by anas-tami**
this is an example for swift

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +69,23 @@ fun main(args: Array<string>) {
...
+```
+var data = "2017/6/25"
+ 
+ if data == "2017/6/25"
+ {
+       print("Eid Mubarak 🎉")  
+ }
+ else
+ {
+ print("Error! 🤓 there is no Eid"
```
Comment (alosefer) [inline README.md:82]: well done, please add ) in the end so it works fine

**Review by alosefer — COMMENTED** [review]
please fix it to approve it, thanks

**Review by anas-tami — COMMENTED** [review]
please fix it

**Review by alosefer — APPROVED** [review]
great work👍🌷

**Pull request merged**